adm/style/ppde_transactions.html

```diff
@@ -140,6 +140,13 @@ <h3>{{ lang('WARNING') }}</h3>
 			<dt><span style="font-weight: bold">{{ lang('PPDE_DT_BOARD_USERNAME') }}{{ lang('COLON') }}</span></dt>
 			<dd>{{ BOARD_USERNAME }}</dd>
 		</dl>
+		<dl>
+			<dt><span style="font-weight: bold">{{ lang('PPDE_DT_CHANGE_BOARD_USERNAME') }}{{ lang('COLON') }}</span><br>{{ lang('PPDE_DT_CHANGE_BOARD_USERNAME_EXPLAIN') }}</dt>
```

Comment:
Please replace the `span` with a `label`.
This will allow to send the focus to the input